docs/metastores.md

```diff
@@ -0,0 +1,99 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+This page documents the general configurations to connect to production database through [EclipseLink](https://eclipse.dev/eclipselink/).
+
+## Polaris Server Configuration
+Configure the `metaStoreManager` section in server configuration `polaris-server.yml` as follows:
+```
+metaStoreManager:
+  type: eclipse-link
+  conf-file: META-INF/persistence.xml
+  persistence-unit: polaris
+```
+`conf-file` points to the EclipseLink configuration file and `persistence-unit` tells which unit from the configuration file to use for database connection.
+
+E.g., `polaris-dev` and `polaris` persistence units are configured in the same configuration file to connect to development database and production database respectively. Updating `persistence-unit` from `polaris-dev` to `polaris` switch from the development to the production.
+
+`conf-file` by default points to the embedded resource file `META-INF/persistence.xml` in `polaris-eclipselink` module.
+To specify an external configuration file,
+1) Place `persistence.xml` into a jar file: `jar cvf /tmp/conf.jar persistence.xml`.
+2) Use `conf-file: /tmp/conf.jar!/persistence.xml`.
+
+## EclipseLink Configuration - persistence.xml
+Configuration file `persistence.xml` is to configure the database connection properties, and the properties vary based on different databases and database setup.
```

Comment:
How about this?

The `persistence.xml` file configures database connection properties, which vary based on the database and its setup.

Reply:
Rephrase to "The configuration file `persistence.xml` is used to set up the database connection properties, which can differ depending on the type of database and its configuration.". How do you think?

Reply:
Fine with that. BTW, is it allowed to have a different config file name other than persistence.xml?

Reply:
Yes. Different file names are allowed but EclipseLink doc always uses persistence.xml (https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm).